### PR TITLE
Feature/oidc playground

### PR DESCRIPTION
**Pull request by oharsta**

### CODE REVIEW

File: roles/oidc-playground-client/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+- name: create apache folder
+  file:
+    path: "/var/www/oidc-playground"
+    owner: apache
```
Comment (quartje) [inline roles/oidc-playground-client/tasks/main.yml]: owner should be root, we don't want to make the web server able to write in this directory

File: roles/oidc-playground-server/handlers/main.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  systemd:
+    daemon_reload: yes
+
+- name: restart oidc-playground
+  service:
+    name: oidc-playground
+    state: restarted
+
+- name: update_ca_trust
+  command: update-ca-trust
```
Comment (quartje) [inline roles/oidc-playground-server/handlers/main.yml]: I don't see this handler used in this role, so it can be removed here

Comment (oharsta) [reply roles/oidc-playground-server/handlers/main.yml]: Done

File: roles/oidc-playground-client/tasks/main.yml (hunk 1 above)
Comment (oharsta) [reply roles/oidc-playground-client/tasks/main.yml]: Done

**Pull request merged**